create-repository/index.js

```diff
@@ -0,0 +1,88 @@
+
+import { Octokit } from "@octokit/action"
+import * as core from '@actions/core';
+
+async function main() {
+    const octokit = new Octokit();
+    const [owner,] = process.env.GITHUB_REPOSITORY.split("/");
+    const name = core.getInput('name');
+    const description = core.getInput('description');
+    const allEnv = ['dev','beta','prod']
+    try {
+        const allTeam = await  octokit.rest.teams.list({
+            org: owner,
+        });
+        const devTeam = allTeam.data.find(d => d.name === 'dev')
+        console.log('Create repository using template  : ', name)
```

Comment:
This log tell me _"I'm creating the new repository using the template with name: xxx"_

I suggest something like:
```suggestion
        console.log(`Creating new reposityry ${name} using pbstck/template`)
```

Reply:
Thanks !